src/main/java/org/folio/circulation/domain/Item.java

```diff
@@ -40,6 +40,8 @@
 
 public class Item {
 
+  private static final String ITEM_STATUS = "status";
```

Comment:
This appears to be a duplicate of the constant in `ItemProperties`

Reply:
Oh, thank you. Catch it!